### PR TITLE
[docs][cloud] Performance Advisor

### PR DESCRIPTION
**Pull request by ddhodge**
New performance advisor page/feature

@netlify /preview/yugabyte-cloud/cloud-monitor/cloud-advisor/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+Connections should be distributed equally across all the nodes in the cluster. Unequal distribution of connections can result in hot nodes, causing higher resource use on those nodes. Excessive workload on a specific node can potentially cause stability issues.
+
+**Fix the problem**
+
+- If a load balancer is used to distribute connections among nodes of cluster, review the configuration of the load balancer. If you are using the YugabyteDB Managed load balancer, contact {{<support-cloud>}}.
+- If you are load balancing in your application or client, review your implementation.
+
+### Query load skew
+
+Advisor scans queries to determine whether some nodes are handling more queries than others. Advisor flags nodes that processed 50% more queries than the other nodes in the past hour.
```
Comment (sushantrmishra) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md:99]: We changed the implementation a bit. 

Advisor flags nodes that processed 50% more queries than the other nodes in the past hour or when the perf advisor scan was performed  last time.

**Review by sushantrmishra — CHANGES_REQUESTED** [review]

**Review by yb-andrew — COMMENTED** [review]
There's also a high CPU usage recommendation type, which can happen independent of CPU skew.

**Review by sushantrmishra — APPROVED** [review]

**Review by yb-andrew — APPROVED** [review]
Looks good.

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+## Recommendations
+
+Performance Advisor provides recommendations for the following issues.
+
+### Index suggestions
+
+Performance Advisor suggests dropping unused indexes to improve write performance and increase storage space. Performance Advisor uses the [pg_stat_all_indexes view](https://www.postgresql.org/docs/11/monitoring-stats.html#PG-STAT-ALL-INDEXES-VIEW) to determine unused indexes. Any index where `idx_scan` is 0 is considered unused.
+
+**Why**
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md:32]: Is this an unfinished sentence?

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+Indexes take up storage space on the same disk volume as the main table. They also increase the size of backups and can add to backup and restore time.
+
+Indexes can slow down some operations. For example:
+
+- When doing an INSERT or DELETE on a table, indexes need to be updated to remain consistent with the data in the main table. An index can also prevent a basic insert or delete from being executed as a single row transaction.
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md:38]: Suggestion:  ... a basic insert or delete operation

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+- When doing an UPDATE on a table, if the modified column is part of the index, then the index must be updated as well.
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md:40]: `UPDATE`

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+**Fix the problem**
+
+Connect to the database and use DROP INDEX to delete the unused indexes.
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md:44]: `DROP INDEX`

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+### Schema suggestions
+
+Advisor scans for indexes that can benefit from using range sharding instead of the default hash sharding.
+
+**Why**
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md:50]: The Why pseudo-headers look weird

**Comment by nchandrappa** [comment]
LGTM.

**Pull request merged**